api/src/pcapi/routes/adage_iframe/offers.py

```diff
@@ -34,3 +35,22 @@ def get_offer(authenticated_information: AuthenticatedInformation, offer_id: int
     )
 
     return serializers.OfferResponse.from_orm(offer)
+
+
+@blueprint.adage_iframe.route("/offers/categories", methods=["GET"])
+@adage_jwt_required
+@spectree_serialize(response_model=serializers.CategoriesResponseModel, api=blueprint.api, on_error_statuses=[404])
```

Comment:
```suggestion
@spectree_serialize(response_model=serializers.CategoriesResponseModel, api=blueprint.api, on_error_statuses=[404])
```

T'es sûr pour la 404 ? 😅 
```suggestion
@spectree_serialize(response_model=serializers.CategoriesResponseModel, api=blueprint.api, on_error_statuses=[404])
```